README.zh-CN.md

```
@@ -0,0 +1,113 @@
+| `loading` | boolean | false | 渲染不确定的加载动画。 |
+| `onClick` | React.MouseEventHandler | - | 单击根 SVG 的处理程序。 |
+| `percent` | number \| number[] | 0 | 进度百分比。数组会渲染多条轨道。 |
+| `prefixCls` | string | `'rc-progress'` | 前缀className。 |
+| `railColor` | string | `'#D9D9D9'` | Rail color. |
+| `railWidth` | number | 1 | 导轨行程宽度。 |
+| `steps` | 数字\| { 计数：数量；间隙：数量 } | - | 以离散步骤渲染进度。 |
+| `strokeColor` | 字符串\|记录<字符串，字符串\|布尔值> \|数组<字符串\|记录<字符串，字符串\|布尔值>> | `'#2db7f5'` | 轨道颜色、渐变对象或每条轨道的颜色。 |
+| `strokeLinecap` | `'round'` \| `'butt'` \| `'square'` | `'round'` | Stroke line cap. |
+| `strokeWidth` | number | 1 | 轨道描边宽度。 |
+| `style` | React.CSSProperties | - | 根样式。 |
+| `styles` | 部分<记录<'根'\| '铁路' | 'track'，React.CSSProperties>> | - | 内部插槽的语义化样式。 |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)